src/scss/includes/app.scss

```diff
@@ -101,3 +101,7 @@ noscript {
     font-size: 24px;
   }
 }
+
+.no_ui .panel_container {
```

Comment:
Can we define all the hidden classes inside à `no_ui.scss` file for instance ?

Reply:
sure